src/core_plugins/kibana/server/lib/__tests__/manage_uuid.js

```diff
@@ -27,24 +27,24 @@ describe('core_plugins/kibana/server/lib', function () {
     const testUuid = 'c4add484-0cba-4e05-86fe-4baa112d9e53';
     let kbnServer;
     let config;
+    let servers;
 
     before(async function () {
-      this.timeout(60000); // sometimes waiting for server takes longer than 10
-
-      kbnServer = kbnTestServer.createServerWithCorePlugins();
-
-      await kbnServer.ready();
+      servers = await kbnTestServer.startTestServers({
```

Comment:
Maybe change line 22 to `import { startTestServers } from '../../../../../test_utils/kbn_server.js';`?